tests/AutoReview/FixerTest.php

```diff
@@ -77,6 +76,16 @@ public function testFixerDefinitions(FixerInterface $fixer)
                 $tokens
             );
             $this->assertTrue($tokens->isChanged(), sprintf('[%s] Sample #%d is not changed during fixing.', $fixer->getName(), $sampleCounter));
+
+            $duplicatedCodeSample = array_search(
+                $sample,
+                array_slice($samples, 0, $sampleCounter),
+                false
+            );
+            $this->assertEmpty(
```

Comment:
Actually it *must* be used. Currently, the branch contains two duplicated code examples (from the referenced PRs, not included here). The test only detects one because in `psr0`, the duplicated example is found at index 0, so `assertEmpty` passes while `assertFalse` will not.

Also, this could be simplified using:

```php
$this->assertNotContains(
    $sample,
    array_slice($samples, 0, $sampleCounter),
    sprintf('[%s] Duplicated code examples detected.', $fixer->getName()),
    false,
    false
);
```

but we would lose the indexes of duplicated examples, so current code is ok as well :)